web/views.py

```
@@ -321,9 +325,9 @@
     # Get top latest 3 leaderboard users
     try:
         top_leaderboard_users, user_rank = get_leaderboard(request.user, period=None, limit=3)
-    except Exception as e:
+    except Exception:
         logger = logging.getLogger(__name__)
-        logger.error(f"Error getting leaderboard data: {e}")
+        logger.error("Error getting leaderboard data", exc_info=True)
```

Ruff (0.11.9)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)